src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts

```diff
@@ -1,5 +1,188 @@
+      browser.wait(
+        Helper.EC.elementToBeClickable(PageHelper.getTableCell(name)),
+        10000,
+        'Could not return to buckets page and load table after editing bucket'
+      );
+
+      // click on edited bucket and check its details table for edited owner field
+      PageHelper.getTableCell(name).click();
+      const element_details_table = element
+        .all(by.css('.table.table-striped.table-bordered'))
+        .first();
+      expect(element_details_table.getText()).toMatch(new_owner);
+    });
+  }
+
+  delete(name) {
+    this.navigateTo();
+
+    // wait for table to load
+    browser.wait(Helper.EC.elementToBeClickable(PageHelper.getTableCell(name)), 10000);
+
+    PageHelper.getTableCell(name).click(); // click on the bucket you want to delete in the table
+    element(by.cssContainingText('.table-actions button.dropdown-toggle', '')).click(); // click toggle menu
+    element(by.cssContainingText('li.delete a', '')).click(); // click delete
+    // wait for pop-up to be visible (checks for title of pop-up)
+    browser
+      .wait(Helper.EC.visibilityOf(element(by.cssContainingText('.modal-title.pull-left', ''))), 5000)
+      .then(() => {
+        const getConfirmationCheckbox = () => element(by.cssContainingText('#confirmation', ''));
+        browser
+          .wait(() => Helper.EC.visibilityOf(getConfirmationCheckbox()), Helper.TIMEOUT)
+          .then(() => {
+            PageHelper.checkCheckbox(getConfirmationCheckbox()).then(() => {
+              //  $('#confirmation').click(); // click confirmation checkbox
+              element(by.cssContainingText('button', 'Delete bucket')).click(); // click delete button
+              // wait for bucket to be deleted
+              browser.wait(
+                Helper.EC.not(Helper.EC.presenceOf(PageHelper.getTableCell(name))),
+                10000,
+                'Timed out waiting for bucket deletion'
+              );
+            });
+          });
+      });
+  }
+
+  invalidCreate() {
+    this.navigateTo('create');
+    expect(PageHelper.getBreadcrumbText()).toEqual('Create');
+
+    const nameInputField = element(by.id('bid')); // Grabs name box field
+
+    // Gives an invalid name (too short), then waits for dashboard to determine validity
+    nameInputField.sendKeys('rq');
+    browser.wait(
+      function() {
+        // Waiting for website to decide if name is valid or not
+        return nameInputField.getAttribute('class').then(function(classValue) {
+          return classValue.indexOf('ng-pending') === -1;
+        });
+      },
+      5000,
+      'Timed out waiting for dashboard to decide bucket name validity'
+    );
+
+    // Check that name input field was marked invalid in the css
+    expect(nameInputField.getAttribute('class')).toContain('ng-invalid');
+
+    // Check that error message was printed under name input field
+    expect(element(by.cssContainingText('.form-group', 'Name')).getText()).toMatch(
```

Comment:
I think this is more specific, the old code was not taking in mind which form-group it was.
```suggestion
    expect(element(by.css('#bid + .invalid-feedback')).getText()).toMatch(
```